### PR TITLE
MUMUP-2286: Search field populated on search results page, and nowhere else

### PR DESCRIPTION
**Pull request by jhanstra**
Goes with https://github.com/UW-Madison-DoIT/angularjs-portal/pull/358.

Now the search field shows the thing you're searching for when you're on that page, doesn't everywhere else.

screen recording 2015-12-10 at 01 01 pm

### CODE REVIEW

File: src/main/webapp/portal/search/controllers.js [hunk 1]
```diff
@@ -33,18 +31,23 @@ define(['angular'], function(angular) {
...
+              //frame app redirect
+              window.location = SEARCH.searchURL + $scope.initialFilter;
           }
-        };
+        }
+      };
+        
+      $scope.$on('$locationChangeStart', function(event) {
+        if ($location.url().indexOf('search') == -1) {
+          $scope.initialFilter = '';
```
Comment (vertein) [inline src/main/webapp/portal/search/controllers.js]: Seems dangerous if we ever change urls, but I guess we're in a lot of trouble when we start changing any of our urls around.

**Comment by vertein** [comment]
:+1:

**Comment by timlevett** [comment]
Seems legit. :+1:

File: src/main/webapp/portal/search/controllers.js (hunk 1 above)
Comment (timlevett) [inline src/main/webapp/portal/search/controllers.js]: Future us can worry about that ;)

**Pull request merged**